plugin/adsk/scripts/USDMenuProc.mel

```diff
@@ -47,9 +47,9 @@ global proc USDMenuProc(string $parent, string $obj)
             string $ls = tolower($s);
             if (`match $ls "true|yes|on|1"` !="")
             {
-                menuItem -label "Edit As Maya Data" -image "edit_as_Maya.png" -command ("mayaUsdEditAsMaya \"" + $obj + "\"");
+                menuItem -label "Edit As Maya Data" -image "edit_as_Maya.png" -command ("{waitCursor -state 1; mayaUsdEditAsMaya \"" + $obj + "\"; waitCursor -state 0;}");
```

Comment:
Hey!  Maybe you can do a guard object in MEL!

Reply:
Please let me know if you know how to do it.

Reply:
I would be very surprised if this would be possible :)  MEL is what it is.